root.sh

```diff
@@ -130,6 +135,7 @@ cmake $SOURCEDIR
       ${CXX11:+-Dcxx11=ON}                                                             \
       ${CXX14:+-Dcxx14=ON}                                                             \
       ${CXX17:+-Dcxx17=ON}                                                             \
+      ${CXX17:+-Dcxx20=ON}                                                             \
```

Comment:
Why keep the `CXXvv` variables? You removed the code that sets them.

If you keep them, shouldn't the last one be `${CXX20:+-Dcxx20=ON}` instead?

Reply:
```suggestion
```